scripts/index.html

```diff
@@ -40,6 +40,14 @@ <h1 id="page-title">
             <button id="join-meeting">Join Meeting</button>
           </fieldset>
         </form>
+        <form id="search-people-legend">
```

Comment:
Please do not include tooling changes as part of this PR

Reply:
I have removed these changes and will include them in a seperate PR once this PR is merged.